src/Recognizer.py

```diff
@@ -29,7 +29,23 @@ def find(self, text, entities=None):
     
 class DOBRecognizer(PatternRecognizer):
     def __init__(self):
-        patterns = [Pattern("DOB", r"(?i)(date of birth|dob)\s?:?\s?(\d{2}/\d{2}/\d{4})", score=1.0)]
+        patterns = [Pattern(
+            "DOB",
+            r"""(?i)(date of birth|dob|birth date)\s?:?\s?(
+                \d{1,2}/\d{1,2}/\d{4}|
+                \d{4}/\d{1,2}/\d{1,2}|
+                \d{1,2}-\d{1,2}-\d{4}|
+                \d{1,2}\.\d{1,2}\.\d{4}|
+                \d{4}\.\d{1,2}\.\d{1,2}|
+                \d{4}-\d{1,2}-\d{1,2}|
+                \d{8}|
+                \d{1,2}\s(Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)\s\d{4}|
+                (Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)\s\d{1,2},\s\d{4}|
+                \d{1,2}\s(January|February|March|April|May|June|July|August|September|October|November|December)\s\d{4}|
+                (January|February|March|April|May|June|July|August|September|October|November|December)\s\d{1,2},\s\d{4}
+            )""",
```

Comment:
```suggestion
            r"(?i)"
            + r"(date of birth|dob|birth date|birth)"
            + r"\s?:?\s?"
            + r"("
                + r"\d{1,2}/\d{1,2}/\d{4}|"
                + r"\d{4}/\d{1,2}/\d{1,2}|"
                + r"\d{1,2}-\d{1,2}-\d{4}|"
                + r"\d{1,2}\.\d{1,2}\.\d{4}|"
                + r"\d{4}\.\d{1,2}\.\d{1,2}|"
                + r"\d{4}-\d{1,2}-\d{1,2}|"
                + r"\d{8}
                + r"\d{1,2}\s(?:Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)\s\d{4}|"
                + r"(?:Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)\s\d{1,2},\s\d{4}|"
                + r"\d{1,2}\s(?:January|February|March|April|May|June|July|August|September|October|November|December)\s\d{4}|"
                + r"(?:January|February|March|April|May|June|July|August|September|October|November|December)\s\d{1,2},\s\d{4}"
            + r")",
```

I think it was doing something weird with the white space with the triple quote """ thing.